### PR TITLE
Lesson3

### PR DESCRIPTION
**Pull request by dmitriy-melnikov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,5 +26,8 @@
     "babel-preset-stage-2": "^6.24.1",
     "babel-register": "^6.26.0",
     "nodemon": "^1.18.7"
+  },
+  "dependencies": {
+    "csvtojson": "^2.0.8"
```
Comment (ahaurylin) [inline package.json]: better use fixed version without `^`, to avoid problems with compatibility

File: src/dirwatcher.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+const log = console.log;
```
Comment (ahaurylin) [inline src/dirwatcher.js]: destructuring can be used

File: src/importer.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+//const path = require('path');
+const csv = require('csvtojson');
+
+class Importer extends EventEmitter {
+    constructor() {
+        super();
+    }
+
+    async _read(filePath) {
+        const jsonArray = await csv().fromFile(filePath);
```
Comment (ahaurylin) [inline src/importer.js]: async/await code should be in try{} catch{} block for error handling

Comment (dmitriy-melnikov) [reply src/importer.js]: done

**Pull request merged**